concent_api/core/subtask_helpers.py

```diff
@@ -23,6 +23,8 @@
 from core.models import PendingResponse
 from core.models import Subtask
 from core.payments import bankster
+from core.payments.bankster import remove_deposit_claim_if_transaction_successed_in_blockchain
```

Comment:
You can remove that import and use `bankster.remove_deposit_claim_if_transaction_successed_in_blockchain` in call below, or remove `bankster` import and import all used functions separately, but I don't know if that's possible.

Reply:
done, import removed and now function is called:
`bankster.remove_deposit_claim_if_transaction_successed_in_blockchain`